pkgs/development/python-modules/cherrypy-cors/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonRelaxDepsHook
+, pytestCheckHook
+, setuptools
+, httpagentparser
+, cherrypy
+}:
+buildPythonPackage rec {
+  pname = "cherrypy-cors";
+  version = "1.7.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-gzhM1mSnq4uat9SSb+lxOs/gvONmXuKBiaD6BLnyEtY=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+    pytestCheckHook
+  ];
+  pythonRelaxDeps = true;
+
+  propagatedBuildInputs = [
+    setuptools
+    httpagentparser
+    cherrypy
+  ];
+
```

Comment:
Could you please add `pythonImportsCheck` for quick check?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters